velox/expression/FunctionSignature.h

```diff
@@ -127,7 +127,8 @@ class FunctionSignature {
       TypeSignature returnType,
       std::vector<TypeSignature> argumentTypes,
       std::vector<bool> constantArguments,
-      bool variableArity);
+      bool variableArity,
+      bool streaming = false);
```

Comment:
This property doesn't belong in FunctionSignature.

Please, see related PR: https://github.com/facebookincubator/velox/pull/9050

Reply:
@mbasmanova Updated. Can you help to review again? Thanks.